### PR TITLE
Prepare 3.0 release

### PR DESCRIPTION
**Pull request by Stormheg**
We've added support for Django 4.0 but haven't made a release yet. When we do, https://github.com/torchbox/django-recaptcha/issues/269 is fixed.

Because we are dropping support for older Python and Django versions (a backwards-incompatible change) it is best to create a new major release following [SemVer](https://semver.org/) conventions.

cc co-maintainers

- @tbrlpld
- @thibaudcolas 
- @atodorov
- @rgs258
- @Andrew-Chen-Wang

### CODE REVIEW

**Comment by Stormheg** [comment]
Looks like I can't merge this without an approving review :(

**Review by tbrlpld — APPROVED** [review]
Looks good to me. Thanks @Stormheg

**Comment by thibaudcolas** [comment]
I wasn’t exactly done with the transfer, would have appreciated a heads’up before we went ahead with this :[ Among other things that were still in progress:

- We still have Praekelt named in the package’s metadata, even though they explicitly told me they didn’t want any involvement with the project at this time. I was still researching which author / maintainer details to add there.
- We have Andrew named as the sole maintainer in the README.
- The project’s translations are out of date.

**Comment by Andrew-Chen-Wang** [comment]
just for the future since it's out of scope now: there is a script in DRF SimpleJWT somewhere by 2ykwang that can help automate the translation compilation. I think it makes sense to have that as a GitHub action with workflow_dispatch and on commit to master.

**Pull request merged**
